src/components/shared/ETHAmount.tsx

```diff
@@ -0,0 +1,44 @@
+import { Tooltip } from 'antd'
+
+import { BigNumber } from 'ethers'
+import { formatWad, parseWad } from 'utils/formatNumber'
+import { betweenZeroAndOne } from 'utils/bigNumbers'
+
+import CurrencySymbol from './CurrencySymbol'
+
+import { CURRENCY_ETH } from 'constants/currency'
+
+import ETHToUSD from './ETHToUSD'
+
+// Eth amount which displays the equiv USD amount in a tooltip on hover
+export default function ETHAmount({
+  amount,
+  precision,
+  padEnd,
+}: {
+  amount?: BigNumber | string
+  precision?: number
+  padEnd?: boolean
+}) {
+  // Account for being passed a string amount or a BigNumber amount
+  const isBetweenZeroAndOne =
+    (BigNumber.isBigNumber(amount) && betweenZeroAndOne(amount)) ||
+    betweenZeroAndOne(parseWad(amount))
+
+  const precisionAdjusted = isBetweenZeroAndOne ? 2 : precision
+
+  const formattedETHAmount = formatWad(amount, {
+    precision: precisionAdjusted ?? 0,
+    padEnd: padEnd ?? false,
+  })
+
+  if (amount) {
+    return (
+      <Tooltip title={<ETHToUSD ethAmount={amount} />}>
+        <CurrencySymbol currency={CURRENCY_ETH} />
+        {formattedETHAmount}
+      </Tooltip>
+    )
+  }
+  return <></>
```

Comment:
fwiw i tested return null here and it seemed to work. no big deal though probably

Reply:
Yeah weird it's working for me now too haha. Changed